libjcat/jcat-bt-proof-private.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <glib-object.h>
+
+GByteArray *
+jcat_rfc6962_hash_leaf(GByteArray *buf);
+
+GByteArray *
+jcat_rfc6962_hash_children(GByteArray *lbuf, GByteArray *rbuf);
+
+/*
+ * Compute a subtree hash for a node on or below the tree's right border. Assumes `proof` hashes
+ * are ordered from lower levels to upper, and `seed` is the initial subtree/leaf hash on the path
+ * located at the specified `index` on its level.
```

Comment:
this description text needs to go in the .c file do that the documentation scanner can pick it up. That's the `/**` thing, e.g.

	/**
	 * jcat_context_get_keyring_path:
	 * @self: #JcatContext
	 *
	 * Gets the local state directory the engines are using.
	 *
	 * Returns: (nullable): path
	 *
	 * Since: 0.1.0
	 **/

All documented here: https://gi.readthedocs.io/en/latest/annotations/gtkdoc.html